internal/config/state.go

```diff
@@ -189,13 +194,18 @@ type API struct {
 	Path string `yaml:"path" json:"path"`
 	// The name of the service config file, relative to the API `path`.
 	ServiceConfig string `yaml:"service_config" json:"service_config"`
+	// The status of the API, one of "new" or "existing".
+	Status string `yaml:"status" json:"status"`
```

Comment:
I think we might need to be start writing marshaller functions for our state file. These are fields we do want present in the request message in docker, but we don't want them committed to the state.yaml file of the repo. I think that leaves us with a couple of options.
 
- On marshalling we ignore these fields.
- We have some sort of cleaning function that strips fields like this
- open to other options too.

Reply:
I think we can ignore this value when writing state to a yaml file.